setup.py

```diff
@@ -1,3 +1,87 @@
-from setuptools import setup
+from setuptools import find_packages, setup
```

Comment:
Curious, what's the benefit of switching to `setup.py` for handling the project metadata instead of the previous `setup.cfg`/`pyproject.toml`?